### PR TITLE
Add vivid, a rust package for $LS_COLOR generation

### PR DESCRIPTION
**Pull request by anjos**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by anjos** [comment]
@conda-forge/help-rust could somebody with powers review this?  It is all green 🎉

File: recipes/vivid/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  version: {{ version }}
+
+source:
+  url: https://github.com/sharkdp/vivid/archive/v{{ version }}.tar.gz
+  sha256: 325f16df916e4192d56d589839be474801ffd3d6a105f4a1a033221570ff6b56
+
+build:
+  number: 0
+  ignore_run_exports:
+    - libstdcxx-ng
```
Comment (xhochy) [inline recipes/vivid/meta.yaml:15]: Why have you added this?

Comment (xhochy) [reply recipes/vivid/meta.yaml:15]: This indicates that you don't actually need the C++ compiler.

Comment (anjos) [reply recipes/vivid/meta.yaml:15]: Ignorance, I guess.  I copied a working recipe.  Should I remove it?

**Comment by anjos** [comment]
@xhochy, thanks for your earlier input.  It is ready for another review.

**Review by xhochy — APPROVED** [review]

**Pull request merged**